pkg/api/serialization_test.go

```diff
@@ -94,6 +95,7 @@ func roundTripSame(t *testing.T, item runtime.Object, except ...string) {
 	codec, err := testapi.GetCodecForObject(item)
 	if err != nil {
 		t.Errorf("unexpected error: %v", err)
+		return
```

Comment:
Oh sorry, didn't realize it wasn't a top level test function. It's fine how it is.